task2/go.mod

```diff
@@ -0,0 +1,3 @@
+module btcwallet
```

Comment:
By convention, module name should include the repo location
`github.com/Konstantsiy/go-internship/wallet`

Reply:
Ok, thanks for the correction.